iNaturalist/src/main/res/layout/default_licenses.xml

```diff
@@ -1,117 +1,122 @@
+            android:layout_marginBottom="5dp"
+            android:background="#D5D5D5" />
 
-    <LinearLayout
-        android:padding="18dp"
-        android:orientation="vertical"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
 
-        <TextView
-            android:id="@+id/description"
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/default_licenses_description"
-            />
+            android:orientation="vertical"
+            android:padding="18dp">
 
-        <RelativeLayout
-            android:id="@+id/more_info"
-            android:layout_width="match_parent"
-            android:layout_height="45dp"
-            android:layout_marginTop="20dp"
-            android:background="@drawable/circular_button_gray"
-            >
-            <LinearLayout
-                android:layout_centerInParent="true"
-                android:orientation="horizontal"
-                android:gravity="center_vertical"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content">
-                <ImageView
-                    android:layout_width="20dp"
-                    android:layout_height="20dp"
-                    android:src="@drawable/ic_open_in_new_black_24dp"
-                    android:tint="#696969"
-                    android:layout_marginRight="5dp"
-                    android:importantForAccessibility="no"
-                    />
-                <TextView
+            <TextView
+                android:id="@+id/description"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/default_licenses_description" />
+
+            <RelativeLayout
+                android:id="@+id/more_info"
+                android:layout_width="match_parent"
+                android:layout_height="45dp"
+                android:layout_marginTop="20dp"
+                android:background="@drawable/circular_button_gray">
+
+                <LinearLayout
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_centerInParent="true"
-                    android:text="@string/more_info"
                     android:gravity="center_vertical"
-                    android:textSize="16sp"
-                    android:textColor="#89000000"
-                    android:textStyle="bold"
-                    />
-            </LinearLayout>
-        </RelativeLayout>
-    </LinearLayout>
+                    android:orientation="horizontal">
 
+                    <ImageView
+                        android:layout_width="20dp"
+                        android:layout_height="20dp"
+                        android:layout_marginRight="5dp"
```

Comment:
Please use `android:layout_marginEnd` here. I know that wasn't the way it was, but we have better RTL coming in a branch and I want to make sure this doesn't get left as `Right` when we merge.

Reply:
The majority of the diff here is from auto-formatting, but I'll clean it up a bit more and update margins to be `start`/`end` instead of `right`/`left` etc.